grug/testing/tests/cron.rs

```diff
@@ -36,6 +41,18 @@ mod tester {
 
         Ok(Response::new().add_message(Message::transfer(job.receiver, job.coin).unwrap()))
     }
+
+    pub fn failing_instantiate(_: MutableCtx, _: Empty) -> StdResult<Response> {
+        Ok(Response::new())
+    }
```

Comment:
instead of this function, i would create a new `mod`:

```rust
mod failing_tester {
    pub fn instantiate() { /* ... */ }

    pub fn cron_execute() { /* ... */ }
}
```

one mod, one function. a lot more logical this way